manuscript/Manuscript.qmd

```diff
@@ -1803,7 +1803,7 @@ However, in this paper we chose to focus on spatial bias.
 Being able to make use of spatially biased data would greatly alleviate data shortage problems in ecosystem condition accounts.
 One way to achieve this is modelling (option *a* in the list above).
 Statistical models can describe the general associations between the sampled data and the context where it was sampled (e.g. a set of environmental variables), and use these relationships to predict and project indicator values to areas that where not originally sampled.
-Depending on the data that goes into these models, they can make very precice and good indicators.
+Depending on the data that goes into these models, they can make very precice and good indicators (REF).
```

Comment:
A ref describing the merrits of predictive models in ecology. Maybe this one: https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/1365-2664.12482

Although it focuses on prediction in time, its mostly the same problems.